addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -249,6 +250,8 @@ def migration_invoice_moves(env):
     openupgrade.logged_query(
         env.cr, query + minimal_where + """
             AND aml.old_invoice_line_id IS NULL
+            AND (rc.anglo_saxon_accounting = FALSE
+                OR rc.anglo_saxon_accounting IS NULL)
```

Comment:
A more elegant way is:
```suggestion
            AND rc.anglo_saxon_accounting IS DISTINCT FROM True
```

Reply:
`a IS DISTINCT FROM b` is doing `((a <> b OR a IS NULL OR b IS NULL) AND NOT (a IS NULL AND b IS NULL))`, thus performan-wise should not be recommended to use that when you have b = True, as you are introducing silly comparisons.

Reply:
That's not true with constants, and the planner avoid the `silly` parts.

Reply:
Good to know! Done.